core/src/main/java/org/elasticsearch/common/settings/SettingsModule.java

```diff
@@ -37,6 +38,8 @@
     private final SettingsFilter settingsFilter;
     private final Map<String, Setting<?>> clusterSettings = new HashMap<>();
     private final Map<String, Setting<?>> indexSettings = new HashMap<>();
+    private static final Predicate<String> NO_TRIBE_PREDICATE =  (s) -> s.startsWith("tribe.") == false || TribeService.TRIBE_SETTING_KEYS.contains(s);
```

Comment:
This got me confused. After some research I got what we're doing (exclude settings of tribe client node). Can we maybe flips this to a positive predicate and call it `TRIBE_CLIENT_NODE_SETTING_PREDICATE` ? I think it's worth the verbosity. 

PS. good catch on the top level Tribe node setting. It's great they are validated.